specs/GetFavicon.md

```diff
@@ -0,0 +1,77 @@
+# Background
+A Favicon is an asset which is a part of every webpage, and typically displayed on each tab. Developers would
+like to have an API which allows them to retrieve the Favicon of a webpage, as well as get an update whenever
+the favicon has changed.
+
+# Description
+We propose a new Webview2 event which would allow developers to get access to the current Favicon of a page, 
+as well as be notified when the favicon changes.
+
+# Examples
+## Win32 C++ Registering a listener for favicon changes
+    ```cpp
+        CHECK_FAILURE(m_webView->add_FaviconChanged(
+            Microsoft::WRL::Callback<ICoreWebView2FaviconUpdateEventHandler>(
+            [this](ICoreWebView2* sender, IUnknown* args) -> HRESULT
+            {
+                
+                LPWSTR value;
+                CHECK_FAILURE(sender->get_FaviconUrl(&value));
```

Comment:
Generally we want the sample code to do something useful. In this case what do we expect the end dev to do with the favicon URI?